File: C/Exercicio8.c
//Algoritmo que lê uma variável numérica e imprime, na condição de que:
//Somente se o número for maior que 100, caso contrário irá imprimir 0.
#include <stdio.h>

int main() {
    //Variáveis
    int numero;

    //Entrada
    printf("Insira um número: \n");
    scanf("%d", &numero);

    //Processamento
    if (numero > 100){
        //Saída
        printf("%d \n", numero);
    }else{
        numero = 0;
    }
    return 0;
}

File: Java/demo/src/main/java/com/exercicios/Exercicio8.java
//Algoritmo que lê uma variável numérica e imprime-a se for maior do que 100.
//Caso não seja, deve imprimir a variável com valor 0.
package com.exercicios;

import java.util.Scanner;

public class Exercicio8 {
    public static void main(String[] args) {   
        //Variáveis
        int num;
        Scanner entrada = new Scanner(System.in);

        //Entrada
        System.out.println("Digite um número: ");
        num = entrada.nextInt();

        //Processamento
        if (num > 100) {
            //Saída
            System.out.println(num);
        }else{
            num = 0;
            //Saída
            System.out.println(num);
        }

    entrada.close();
    }   
}